samples/boards/96b_argonkey/microphone/src/main.c

```diff
@@ -51,38 +51,38 @@ void signal_sampling_started(void)
 	static struct device *led0, *led1;
 
 	led0 = device_get_binding(DT_ALIAS_LED0_GPIOS_CONTROLLER);
-	gpio_pin_configure(led0, DT_ALIAS_LED0_GPIOS_PIN, GPIO_DIR_OUT);
-	gpio_pin_write(led0, DT_ALIAS_LED0_GPIOS_PIN, 1);
+	gpio_pin_configure(led0, DT_ALIAS_LED0_GPIOS_PIN, GPIO_OUTPUT);
```

Comment:
We need to do `GPIO_OUTPUT | DT_ALIAS_LED0_GPIOS_FLAGS` or otherwise the logical levels defined in DTS won't be honored. Since the LEDs on 96b_argonkey are active high it works anyway but if someone will copy/paste the code to use on another board it won't.

Reply:
@mnkp 
Correct!
I realized it when doing #20941, but I forgot to change this PR.